### DISCUSSION TITLE
Does this support Raspberry Pi?

### DISCUSSION THREAD

**Discussion by Jamesking56** [shivammathur/setup-php#480] [Q&A / Help] [answered]
Hello,

We as a business are looking to replace a VPS we pay monthly with a cloud provider for to run GitHub self-hosted runners with an internal high-specced Raspberry Pi.

Does this action run fine on a Raspberry Pi? Has anyone else tried this?

Thanks.

**Comment by shivammathur** [comment, marked as answer]
@Jamesking56 
If you would be using Ubuntu on your pi, it should work.
If using Raspbian, wait for the next release as that will add support for Debian-based OS.

**Reply by Jamesking56** [reply]
We would use Ubuntu, we use it across the board as a business - good to know this will work though 👍
